lib/galaxy/tools/data/__init__.py

```diff
@@ -265,9 +265,8 @@ def add_entry(self, entry, allow_duplicates=True, persist=False, persist_on_erro
         return self._update_version()
 
     def add_entries(self, entries, allow_duplicates=True, persist=False, persist_on_error=False, entry_source=None, **kwd):
-        if entries:
-            for entry in entries:
-                self.add_entry(entry, allow_duplicates=allow_duplicates, persist=persist, persist_on_error=persist_on_error, entry_source=entry_source, **kwd)
+        for entry in entries:
```

Comment:
Is `entries` always going to be iterable ?

Reply:
found it, answer should be yes.

Reply:
`add_entries()` is only called once, with `other_table.data` as parameter. And `TabularToolDataTable.data` is initialised to `[]`, so yes.